Assets/Scripts/Combat/CreatureStats.cs

```diff
@@ -147,8 +135,7 @@ private int BonusRearDamage()
 
     private void DisplayPopup(string text)
     {
-        textLabelTimer = 5.0f;
-        textLabelText = text;
+        PopupTextController.CreatePopupText(text, transform);
```

Comment:
Might look better if the popup texts are a little less verbose (at minimum, get rid of "inflicted").

Reply:
Done